secant/Utils/Logging/LogStore.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  LogStore.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 23.01.2023.
+//
+
+import Foundation
+import OSLog
+
+extension Date {
+    func timestamp() -> String {
+        let dateFMT = DateFormatter()
```

Comment:
We should cache instance of DateFormatter somehow. Creating DateFormatter is an expensive operation. And `timestamp()` is called for each log. Sorry for missing this on the first review.

Reply:
resolved